source/plugins/languages/analyzer/analyzer.mjs

```diff
@@ -86,21 +87,23 @@ export class Analyzer {
   /**Clone a repository */
   async clone(repository) {
     const {repo, branch, path} = this.parse(repository)
-    let url = /^https?:\/\//.test(repo) ? repo : `https://github.com/${repo}`
+    const token = core.getInput("token")
```

Comment:
Can you gate this behind a `if (process.env.GITHUB_ACTIONS)` so this only execute on GitHub Actions ?
The web instances probably doesn't need to expose its token on guest tokens (even though this indepth mode shouldn't be enabled on them anyways)

Reply:
Yes absolutely. Just pushed the change